python/ccf/proposal_generator.py

```diff
@@ -336,6 +338,31 @@ def set_recovery_threshold(threshold: int, **kwargs):
     return build_proposal("set_recovery_threshold", threshold, **kwargs)
 
 
+@cli_proposal
+def update_root_ca_cert(cert_name, cert_path, skip_checks=False, **kwargs):
+    with open(cert_path) as f:
+        cert_pem = f.read()
+
+    if not skip_checks:
+        try:
+            cert = x509.load_pem_x509_certificate(
+                cert_pem.encode(), crypto_backends.default_backend()
+            )
+        except Exception as exc:
+            raise ValueError("Cannot parse PEM certificate") from exc
+
+        try:
+            oid = x509.ObjectIdentifier("1.2.840.113556.10.1.1")
```

Comment:
Like most non-trivial literals, this would benefit from being a named, package-scope variable. The name would document it, and setting it at the package scope means users can overrides if they wish to without having to modify the code.